placementApp/serializers.py

```diff
@@ -13,18 +14,38 @@ class Meta:
         model = Student
         fields = (
             "id",
-            "first_name",
-            "last_name",
+            "f_name",
+            "l_name",
             "email",
             "sap_ID",
-            "division",
             "department",
             "year",
             "pointer",
             "password",
+            "password2"
         )
 
 
+class CoordinatorSignupSerializer(serializers.ModelSerializer):
+    password = serializers.CharField(
+        write_only=True,
+        required=True,
+        style={'input_type': 'password'},
+    )
+    password2 = serializers.CharField(style={'input_type': 'password'}, write_only=True)
+    class Meta:
+        model = Coordinator
+        fields = (
+            "id",
+            "f_name",
+            "l_name",
+            "email",
+            "department",
+            "password",
+        )
+
+
+
```

Comment:
In StudentSerializer change the field from first_name to f_name and last_name to l_name